tmp/cache/assets/sprockets/v3.0/-9/-9SbmgsxtG2y9LPS_QbIKZ0pnAtzpni78pVk1towk68.cache

```diff
@@ -0,0 +1 @@
+"%�<������X���̀#+�����,�(o���
```

Comment:
Remainder of these fils should be in your `.gitignore`. You can ignore whole directories, which should help. Be sure to use `git status` to check to see what you're committing before you push.